### ISSUE TITLE
Metrics as methods of models

### ISSUE THREAD

**Issue by lars-reimann** [Safe-DS/Stdlib#64]
### Is your feature request related to a problem?

In some sense metrics are a property of models (in connection with some data).

### Desired solution

Think about whether it makes sense to have them as methods on models. This should be easy to implement once #60 is done.

### Additional Context (optional)

If this gets implemented, #63 is no longer needed.
